scss/dijit/form/_common.scss

```diff
@@ -30,6 +30,12 @@
   position: relative;
 }
 
+// IE fix
+.dijitSelect .dijitArrowButtonContainer {
```

Comment:
Can't do... need to find a way to make this auto adjust... if we change the height of the field this will break, also this breaks the 32px in all the fields:

<img width="233" alt="screen shot 2017-03-23 at 6 08 56 pm" src="https://cloud.githubusercontent.com/assets/751424/24275259/db213b40-0ff3-11e7-9820-dd30bbac1f9e.png">